docs/tools/list.md

```diff
@@ -0,0 +1,51 @@
+# ADK Tools list
+
+Looking for a pre-built [ADK Tool](/adk-docs/tools/) to use in an ADK agent 
+workflow? Here is a list of available ADK Tools:
+
+### ADK-provided tools
+
+*   **[Google Search](/adk-docs/tools/built-in-tools/#google-search)**:
+    Perform web searches using Google Search.
+*   **[Code Execution](/adk-docs/tools/built-in-tools/#code-execution)**:
+    Execute code using Gemini 2 models.
+
+### Google Cloud tools
+
+*   **[GKE Code Executor](/adk-docs/tools/built-in-tools/#gke-code-executor)**:
+    Run AI-generated code in a secure and scalable GKE Sandbox environment.
+*   **[Vertex AI Search](/adk-docs/tools/built-in-tools/#vertex-ai-search)**:
+    Search across your private, configured data stores in Vertex AI Search.
+*   **[Vertex AI RAG Engine](/adk-docs/tools/built-in-tools/#vertex-ai-rag-engine)**:
+    Perform private data retrieval using Vertex AI RAG Engine.
+*   **[BigQuery Tools](/adk-docs/tools/built-in-tools/#bigquery)**:
+    Connect with BigQuery to retrieve data and perform analysis.
+*   **[Spanner Tools](/adk-docs/tools/built-in-tools/#spanner)**:
+    Interact with Spanner to retrieve data, search, and execute SQL.
+*   **[Bigtable Tools](/adk-docs/tools/built-in-tools/#bigtable)**:
+    Interact with Bigtable to retrieve data and and execute SQL.
+*   **[MCP Toolbox for Databases](/adk-docs/tools/google-cloud-tools/#toolbox-tools-for-databases)**:
+    Connect a wide variety of databases to your agents.
+*   **[Apigee API Hub Tools](/adk-docs/tools/google-cloud-tools/#apigee-api-hub-tools)**:
+    Turn any documented API from Apigee API hub into a tool.
```

Comment:
Add any API and it's documentation into an agent toolset, with Apigee API Hub, and optimize tool descriptions with SpecBoost.

(Note, will expose MCP Server eventually)